ticker/src/main/java/com/robinhood/ticker/TickerColumn.java

```diff
@@ -122,9 +124,19 @@ private void setCharacterIndices() {
     }
 
     void onAnimationEnd() {
+        checkForDrawMetricsChanges();
         minimumRequiredWidth = currentWidth;
     }
 
+    private void checkForDrawMetricsChanges() {
+        final float currentTargetWidth = metrics.getCharWidth(targetChar);
```

Comment:
is this expensive to compute? worth caching?

Reply:
Can't actually cache this because the `TickerColumn` class doesn't know when the drawmetrics gets invalidated, but its also not expensive to compute (just a map lookup).